Add property: "delimiter" in seed-config

kouta-fujimura opened the pull request:
https://docs.getdbt.com/reference/resource-configs/delimiter

dbt supports delimiter from v1.7, but currenly not exists in two files.
- dbt_project.json
- dbt_yml_files.json

dave-connors-3 reviewed (changes requested):
hey @kouta-fujimura ! thanks for the PR -- generally we are only amending the jsonschemas under the `latest/` directory, but since you did the work here, I'm happy to merge these in to 1.7 as well! 

It looks like we'll need to also edit  `latest/dbt_project-latest.json` to make sure there's support for `delimiter` there as well. Additionally, if you could revert the change to the `dbt_project.json` at the root of the repo, that would be great! those are legacy files that we don't update.

kouta-fujimura commented:
@dave-connors-3 
thank you for the review !
sorry, I had mistakenly mixed up `/dbt_project.json` and `/latest/dbt_project-latest.json`, so I added a commit to fix it (Additionally, I have added a property without plusses for CI).

Phil-Garmann commented:
Any progress on this PR? 🤞
